docs/.vuepress/config.js

```diff
@@ -439,17 +430,22 @@ module.exports = {
     ],
     [
       'vuepress-plugin-sitemap',
-      DEPLOY_DOMAIN
-        ? {
-            hostname: DEPLOY_DOMAIN
-          }
-        : false
+      {
+        hostname: DEPLOY_DOMAIN
+      }
     ],
     [
-      'robots',
+      'vuepress-plugin-robots',
       {
         host: DEPLOY_DOMAIN
       }
+    ],
+    [
+      'vuepress-plugin-container',
+      {
+        type: 'callout',
+        defaultTitle: ''
+      }
```

Comment:
this plugin is enabled in the parent theme so we shouldn't require it here too?

Reply:
to create a new container we need to add another array here

Reply:
https://github.com/vuepress/vuepress-plugin-container#usage

Reply:
sorry, what I was meaning is could we reuse one of the existing blocks? https://vuepress.vuejs.org/guide/markdown.html#custom-containers

Reply:
just noticed they print the type too so wouldn't be suitable. ok - let's go this route!